egs2/mucs21_subtask1/asr1/local/down_sample.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+DIR=$1
+reDir=$2
+
+for i in $DIR; do
```

Comment:
since this script is very simple, how about directly present it in `egs2/mucs21_subtask1/asr1/local/prepare_data.sh`